test/e2e/fixture-builder.js

```diff
@@ -486,15 +486,15 @@ class FixtureBuilder {
 
   withPreferencesControllerAdditionalAccountIdentities() {
     return this.withPreferencesController({
-      identites: {
+      identities: {
         '0x5cfe73b6021e818b776b421b1c4db2474086a7e1': {
           address: '0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
           lastSelected: 1665507600000,
           name: 'Account 1',
         },
         '0x09781764c08de8ca82e156bbf156a3ca217c7950': {
           address: '0x09781764c08de8ca82e156bbf156a3ca217c7950',
-          lastSelected: 1665507500000,
+          lastSelected: 1665507800000,
```

Comment:
Changed to maintain the order now that the fixture data is loaded correctly.

See: https://github.com/MetaMask/metamask-extension/pull/25291/files#r1639256555